README.md

```diff
@@ -357,9 +359,18 @@ fetch('/api/authentication/nonce', {
 
 
 ### Configuring Your Project
-TODO: Define required and optional configuration variables for project's `settings.py`
-
 
+#### django-siwe-auth specific options in settings.py:
```

Comment:
Please separate these options out into two sections: django-siwe-auth options and django-native options. I want to differentiate between which configurations are custom from django-siwe-auth.

Django Native: AUTH_USER_MODEL, AUTHENTICATION_BACKENDS, LOGIN_URL, SESSION_COOKIE_AGE
Django SIWE Auth: CREATE_GROUPS_ON_AUTHN, CREATE_ENS_PROFILE_ON_AUTHN, CUSTOM_GROUPS, PROVIDER